circuits/circuits/mint.circom

```diff
@@ -1,20 +1,63 @@
 pragma circom 2.1.5;
 
 include "./node_modules/circomlib/circuits/comparators.circom";
-
+include "./merkle_tree.circom";
 
 // If the sender is the 0 address, the transaction is a mint request,
 // if the recipient is 0, the transaction is a withdrawal, if neither are
 // 0 it's an L2->L2 send, and both can't be 0.
-template Mint(levels) {
+template Mint(levels, mintLevels) {
     signal input sender;
     signal input recipient;
+
+    signal input mint_root;
```

Comment:
This is good for now, but in the future we might be able to pass the mint request directly into circuit as an input. The main reason to do this is the gas cost of adding requests into the mint merkle tree onchain.

Reply:
sounds good. we can add a _build mint tree_ circuit later.